### DISCUSSION TITLE
Cannot send email to external source!

### DISCUSSION THREAD

**Discussion by nabilrt** [docker-mailserver/docker-mailserver#3771] [Q&A]
Hello!, I have configured docker-mailserver, now i can send email to domain to domain such as info@mail.example.com to info1@mail.example.com, but i cannot send email to @gmail.com or other domains.

my docker-compose.yml
version: '2.0'

services:
  mailserver:
    image: docker.io/mailserver/docker-mailserver:latest
    container_name: mailserver
    # If the FQDN for your mail-server is only two labels (eg: example.com),
    # you can assign this entirely to `hostname` and remove `domainname`.
    hostname: mail.docacamedia.com # replace with your domain
    domainname: mail.docacamedia.com # replace with your domain
    dns: 1.1.1.1 # This line is newly added and recommended
    env_file: mailserver.env
    # More information about the mail-server ports:
    # https://docker-mailserver.github.io/docker-mailserver/latest/config/security/understanding-the-ports/
    # To avoid conflicts with yaml base-60 float, DO NOT remove the quotation marks.
    ports:
      - "25:25"    # SMTP  (explicit TLS => STARTTLS)
      - "143:143"  # IMAP4 (explicit TLS => STARTTLS)
      - "465:465"  # ESMTP (implicit TLS)
      - "587:587"  # ESMTP (explicit TLS => STARTTLS)
      - "993:993"  # IMAP4 (implicit TLS)
    volumes:
      - ./docker-data/dms/mail-data/:/var/mail/
      - ./docker-data/dms/mail-state/:/var/mail-state/
      - ./docker-data/dms/mail-logs/:/var/log/mail/
      - ./docker-data/dms/config/:/tmp/docker-mailserver/
      - /etc/localtime:/etc/localtime:ro
      - /etc/letsencrypt/:/tmp/ssl:ro # Add this line. It will be used later.
    restart: always
    stop_grace_period: 1m
    cap_add:
      - NET_ADMIN
      
![image](https://github.com/docker-mailserver/docker-mailserver/assets/67606812/34438b7d-1775-44c1-a146-221dbdafe3d0)

**Comment by casperklein** [comment]
I don't think this issue is DMS related. Probably a local host firewall or your internet provider is blocking port 25.

You can manually test this either from your host or the container with: `nc -C 142.250.123.26 25`

**Reply by nabilrt** [reply]
can i send with port 587 instead of 25? If so, do you have any tip fo that?

**Reply by casperklein** [reply]
No, port 25 is mandatory.

**Reply by nabilrt** [reply]
Can you help me with a guide to unblock port 25 on ubuntu 20.04? 
I have a postfix-main.cf file with the following config
mydestination = localhost.$mydomain, localhost

Referring to this issue, it solved my internal mail sending and recieving email from external source issues. But i still can't send email to external sources such as Gmail etc.
#1278

**Reply by casperklein** [reply]
Unless you configured it, there should be nothing blocking outbound traffic in Ubuntu. It's more likely, that your ISP blocks the traffic.

**Comment by BozhanL** [comment]
It seems that your email server is hosted in Digital Ocean, which [blocks port 25](https://docs.digitalocean.com/support/why-is-smtp-blocked/)
